board/views.py

```diff
@@ -18,7 +19,14 @@ class Game:
                  '../../static/board/img/screen3.png',
                  '../../static/board/img/screen1.png', '../../static/board/img/screen2.jpg',
                  '../../static/board/img/screen3.png']
+    genres = ["Arcade", "Action"]
+    platforms = ['PC', 'PS4']
+    rating = {'users': [7.32, 123], 'critics': [7.11, 123]}
     id = 1
+    tweets = [['tweet_name', "Pac-Man is a maze arcade game developed and released by Namco in 1980."
+                "The original Japanese title of Puck Man was changed to Pac-Man for international releases as "
+                "a preventative",
+               datetime.datetime.now()]] * 5
```

Comment:
Inconsistent use of `"` vs `'`. Consider using either one or another. Also a good idea is to use linting for your code: https://www.pylint.org/